test/fp.functorify.js

```diff
@@ -10,14 +10,15 @@ const tripleFnInCb = callback =>
 const asyncMultBy11 = (x, callback) =>
   process.nextTick(() => callback(null, x * 11));
 
-metatests.test('asAsync all functions test', test => {
+metatests.test('functorify all methods test', test => {
+  // prettier-ignore
```

Comment:
Why, though?

Reply:
No, I'm referring to the `prettier-ignore` directive.

Reply:
It's because of suggested by prettier formatting in 20-22 lines. 
@tshemsedinov pointed it out here.
https://github.com/metarhia/metasync/pull/311#discussion_r251081264
Prettier suggests this formatting
```javascript
    .ap(tripleFnInCb)((err, res) => {
    test.error(err);
    test.strictSame(res, (3 + 5) * 7 * 3 * 11);
    test.end();
    });
```